subsys/bluetooth/mesh/Kconfig

```
@@ -694,10 +694,19 @@ config BT_MESH_SEQ_STORE_RATE
 	  off with a value that's guaranteed to be larger than the last
 	  one used before power off.
 
+config BT_MESH_RPL_STORE_MANUAL
+	bool "Manually store RPL in persistent storage"
+	help
+	  When this option is enabled, the RPL entries can only be stored
+	  manually through bt_mesh_rpl_pending_store(). Note that in case
+	  of IV Index update the outdated RPL entries will still be stored
+	  by BT_MESH_STORE_TIMEOUT.
+
 config BT_MESH_RPL_STORE_TIMEOUT
 	int "Minimum frequency that the RPL gets updated in storage"
```

What is the unit here? How about?
```suggestion
	int "Minimum interval after which unsaved RPL entries are written to the flash"
```

---

I am partially agree with this suggestion. Units are described in `help` and it is not not necessary that storage is flash.